packages/gluestick/src/commands/__tests__/start.test.js

```diff
@@ -0,0 +1,102 @@
+/* @flow */
```

Comment:
What is the point of using `flow` if then we `FlowIgnore` everything ? Is because you don't want to write all options that `Context`, `GSConfig`, etc have? Once we write them once, when we can reuse it in every test. Also, can you just use/import `gluestickConfig.js`? I think also @dratwas wrote some mocks, we can reuse them.

Flow is a low level testing tool too. So if we ensure types (not ignoring them), we can avoid some test cases (repetitive).